.github/CODEOWNERS

```diff
@@ -9,3 +9,19 @@
 /master   @determined-ai/backend
 /agent    @determined-ai/backend
 /proto    @determined-ai/backend
+
+# Harness is mapped to ml-sys by default, except for `cli`, `deploy`,
+# and auto-generated bindings and version files.
+/harness  @determined-ai/ml-sys
+/harness/determined/deploy
+/harness/determined/cli
+/harness/tests/cli
+/harness/determined/common/api/bindings.py
+/harness/determined/__version__.py
+
+/schemas                          @determined-ai/ml-sys
+/bindings/generate_bindings_py.py @determined-ai/ml-sys
+/e2e_tests/tests/*.py             @determined-ai/ml-sys
```

Comment:
`e2_tests` is in a rough place. I think it needs an owner, and I support your suggestion that that owner be ml-sys. If it does turn out to be ml-sys, we should schedule some time in the next quarter to clean it up and add a dev guide for it.

Reply:
@MikhailKardash is looking into pruning the examples, which might be related to cleaning up `e2e_tests`.